package androidx.compose.material.icons.outlined;

import bj.n;
import g1.m0;
import g1.t;
import j0.b;
import k0.a;
import k0.c;
import k1.e;
import k1.f;
import k1.h0;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class TodayKt {
    private static f _today;

    public static final f getToday(b bVar) {
        l.f("<this>", bVar);
        f fVar = _today;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Outlined.Today", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        m0 m0Var = new m0(t.f6905b);
        n nVarB = c.b(19.0f, 3.0f, -1.0f, 18.0f, 1.0f);
        k0.b.r(nVarB, -2.0f, 2.0f, 8.0f, 3.0f);
        nVarB.l(8.0f, 1.0f);
        nVarB.l(6.0f, 1.0f);
        nVarB.t(2.0f);
        nVarB.l(5.0f, 3.0f);
        nVarB.i(-1.11f, 0.0f, -2.0f, 0.9f, -2.0f, 2.0f);
        nVarB.t(14.0f);
        nVarB.i(0.0f, 1.1f, 0.89f, 2.0f, 2.0f, 2.0f);
        nVarB.k(14.0f);
        nVarB.i(1.1f, 0.0f, 2.0f, -0.9f, 2.0f, -2.0f);
        nVarB.l(21.0f, 5.0f);
        nVarB.i(0.0f, -1.1f, -0.9f, -2.0f, -2.0f, -2.0f);
        a.n(nVarB, 19.0f, 19.0f, 5.0f, 19.0f);
        a.y(nVarB, 5.0f, 9.0f, 14.0f, 10.0f);
        nVarB.n(19.0f, 7.0f);
        nVarB.l(5.0f, 7.0f);
        a.y(nVarB, 5.0f, 5.0f, 14.0f, 2.0f);
        a.p(nVarB, 7.0f, 11.0f, 5.0f, 5.0f);
        nVarB.l(7.0f, 16.0f);
        nVarB.g();
        e.a(eVar, nVarB.f3166a, 0, m0Var);
        f fVarB = eVar.b();
        _today = fVarB;
        return fVarB;
    }
}
